docs/howto/custom-template-tags.txt

```diff
@@ -498,6 +498,127 @@ you see fit:
     {% current_time "%Y-%m-%d %I:%M %p" as the_time %}
     <p>The time is {{ the_time }}.</p>
 
+.. _howto-custom-template-tags-simple-block-tags:
+
+Simple block tags
+-----------------
+
+.. method:: django.template.Library.simple_block_tag()
+
+Passing arguments may not be enough for some usages -- you may need to pass chunks of
+rendered template into your custom tags. This can be achieved using the
+``simple_block_tag`` helper function of ``django.template.Library``.
+``simple_block_tag`` functions similar to :meth:`~django.template.Library.simple_tag()`,
+however additionally passes a ``content`` argument which contains its rendered template
+content.
+
+For example, a custom block tag which created a chart could look like this::
+
+    from django import template
+    from myapp.charts import render_chart
+
+    register = template.Library()
+
+
+    @register.simple_block_tag
+    def chart(content):
+        return render_chart(source=content)
```

Comment:
Something which occured to me, is that the use-cases of a tag like this are fairly limited. However, a variant which accepts a template and processes context (akin to `inclusion_tag`) could be incredibly helpful. Most of the building blocks are here to make that happen, but not sure if that warrants a separate PR, or it's worth trying to squeeze in here? I'm not sure what the diff size would be, which might be the deciding factor.

Reply:
I've tried this out, and the diff is quite small. `InclusionNode` can be subclassed in much the same way as was done with `SimpleNode`, with similar duplication of `inclusion_tag`. 

With that said, there's little overlap between the 2 feature, so it probably does make sense they be separate. I can contribute those once this is merged.